NMS-15426:  Geographical Map map search capability is not as described in the docs (#6024)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/geographical-map/searching.adoc

## Search fields

> This search uses the Search REST endpoint that can find nodes by any of the fields this API offers.
> This includes the previous supported fields listed.

Reviewer: ```suggestion
The search function for the geographical map uses the Search REST endpoint.
This endpoint can find nodes using any of the fields offered by the API, including the previously listed fields:
```

[options="header"]
[cols="3*"]
|====
| Field
| Description

> | Allow substring

Reviewer: ```suggestion
| Allow Substring
```

| nodeId
| The ID of the node
| No

| nodeLabel
| A text description
| Yes

| foreignSource

> | The provisioned foreign source, if any

Reviewer: ```suggestion
| The provisioned foreign source, if any.
```

| No

| foreignId

> | The provisioned foreign ID, if any

Reviewer: ```suggestion
| The provisioned foreign ID, if any.
```

| No

| ipAddress
| The primary IP address
| Yes

| severityLabel

> | The severity (Normal, Warning, etc.)

Reviewer: ```suggestion
| The severity (Normal, Warning, and so on)
```

| No

| description
| The description field from the node's assets
| Yes

| maintContract
| The maintenance contract from the node's assets
| Yes

| category

> | Match a category, if any

Reviewer: ```suggestion
| Match a category, if any.
```

| Yes

|====

Note that all searches are case-insensitive. Also note this search currently only allows node selection by the any of the search fields in the list.

> ## Exact and substring matching example

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> To search for en exact or a partial text in one of the fields, enter the desired text.

Reviewer: ```suggestion
If you want to search for exact or partial text in one of the search fields, simply type the desired text in the appropriate field.
```

> 

Reviewer: ```suggestion
```
